monitoring/angle-monitoring/src/main/java/com/farao_community/farao/monitoring/angle_monitoring/RaoResultWithAngleMonitoring.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.monitoring.angle_monitoring;
+
+import com.farao_community.farao.commons.FaraoException;
+import com.farao_community.farao.commons.Unit;
+import com.farao_community.farao.data.crac_api.Instant;
+import com.farao_community.farao.data.crac_api.RemedialAction;
+import com.farao_community.farao.data.crac_api.State;
+import com.farao_community.farao.data.crac_api.cnec.AngleCnec;
+import com.farao_community.farao.data.crac_api.network_action.NetworkAction;
+import com.farao_community.farao.data.rao_result_api.ComputationStatus;
+import com.farao_community.farao.data.rao_result_api.RaoResult;
+import com.farao_community.farao.data.rao_result_api.RaoResultClone;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * class that enhances rao result with angle monitoring results
+ * @author Mohamed Ben Rejeb {@literal <mohamed.ben-rejeb at rte-france.com>}
+ */
+public class RaoResultWithAngleMonitoring extends RaoResultClone {
+
+    private final RaoResult raoResult;
+    private final AngleMonitoringResult angleMonitoringResult;
+
+    public RaoResultWithAngleMonitoring(RaoResult raoResult, AngleMonitoringResult angleMonitoringResult) {
+        super(raoResult);
+        this.raoResult = raoResult;
+        this.angleMonitoringResult = angleMonitoringResult;
+    }
+
+    @Override
+    public ComputationStatus getComputationStatus() {
+        if (angleMonitoringResult.isSecure() || angleMonitoringResult.isUnsecure()) {
+            return raoResult.getComputationStatus();
+        } else {
+            return ComputationStatus.FAILURE;
+        }
```

Comment:
for sake of functional redability, I would prefer this:
```suggestion
        if (angleMonitoringResult.isDivergent() || angleMonitoringResult.isUnknown()) {
            return ComputationStatus.FAILURE;
        } else {
            return raoResult.getComputationStatus();
        }
```